lxde-base/lxdm/lxdm-0.4.1-r10.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="2"
```

Comment:
Do you specifically need this version? If not, I would not patch this version, wait for 0.5.3-r1 to get stabilized, and this version to get cleaned up.